go.mod

```diff
@@ -48,6 +48,7 @@ require (
 	github.com/spf13/cobra v1.9.1
 	github.com/spf13/pflag v1.0.6
 	github.com/stretchr/testify v1.10.0
+	github.com/yuin/goldmark v1.7.8
```

Comment:
We already hah `yuin/goldmark` as an *indirect* dependency. It's now considered as a *direct* one, since it's been used in the tests.